test/scripts/filters/meta_generator.js

```diff
@@ -32,4 +33,22 @@ describe('Meta Generator', () => {
     const resultType = typeof result;
     resultType.should.eql('undefined');
   });
+
+  it('ignore empty head tag', () => {
```

Comment:
Other tests need to be updated, as the filter no longer seeks `<title>`.

Reply:
Updated.